File: src/Campaigns/Group/send.js
import axios from 'axios';

const data = {
    title: "Announcement",
    sender: "VELSTACK",
    group_id: "d565d047-df83-4b33-a498-f9de2aca50e6",
    message: 'First API message from Velstack'
}

axios.post('https://sms.velstack.com/api/messaging/group/sms', data, {
    headers: {
        Authorization: "Bearer " + "API_KEY",
        Accept: "application/json",
    }
}).then((response) => {
    alert('Message sent successfully')

}).catch((errors) => {
    alert(errors.response.data.message)
})

File: src/Campaigns/Quick/send.cpp
  CURL *hnd = curl_easy_init();

  curl_easy_setopt(hnd, CURLOPT_CUSTOMREQUEST, "POST");
  curl_easy_setopt(hnd, CURLOPT_URL, "https://sms.velstack.com/api/quick/sms");

  struct curl_slist *headers = NULL;
  headers = curl_slist_append(headers, "accept: application/json");
  headers = curl_slist_append(headers, "Authorization: Bearer API_KEY")
  curl_easy_setopt(hnd, CURLOPT_HTTPHEADER, headers);

  curl_easy_setopt(hnd, CURLOPT_POSTFIELDS,
      "title=string&information&sender_id=string&velstack&message=string&message_content&recipient=string&0205550368");

  CURLcode ret = curl_easy_perform(hnd);